include/sys/sysinfo.h

```diff
@@ -0,0 +1,78 @@
+/****************************************************************************
+ * include/sys/sysinfo.h
```

Comment:
why has this change?

Reply:
I accidentally polluted the commit history, I'll try to fix it now